### PR TITLE
add router progress bar

### PR DESCRIPTION
**Pull request by avneesh0612**
# Description

This adds a router progress bar, which indicates the user on switching pages.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
It was a simple test. I just navigated through the pages

# Checklist:

- [] My code follows the [style guidelines](https://github.com/atapas/webapis-playground/blob/master/HOW-TO-ADD-DEMO.md) of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by atapas** [comment]
@avneesh0612 this is great. Please create an issue to track and link this one. I shall assign that to you!

**Comment by avneesh0612** [comment]
Awesome @atapas . I have opened it- #77

File: pages/_app.js [hunk 1]
```diff
@@ -22,6 +23,8 @@ function MyApp({ Component, pageProps, router }) {
         },
       }}
     >
+      <NextNProgress height={3} color="#fff" />
```
Comment (niteshseram) [inline pages/_app.js:26]: The spinner at top right side looks bad. Is there a way to remove it?

Comment (avneesh0612) [reply pages/_app.js:26]: Yes we can, we just need to add this instead @niteshseram -

``` 
<NextNProgress height={3} color="#fff" options={{ showSpinner: false }} />
```

Comment (avneesh0612) [reply pages/_app.js:26]: So should I disable the spinner? @niteshseram

Comment (atapas) [reply pages/_app.js:26]: @avneesh0612 Yes remove that.. as in it is hardly seen and when it is seen it is not serving much. IMO we can remove the spinner at the top right.

Comment (avneesh0612) [reply pages/_app.js]: okay @atapas removed it.

**Review by atapas — APPROVED** [review]
@avneesh0612 Changes are looking good to me. Thank You!

**Comment by atapas** [comment]
@avneesh0612 

Thanks a lot for taking the issue and making the app better. Please feel free to pick up more, contribute more. I wish you a great #Hactoberfest ahead. You will be added to the contribution list right after we merge this PR. Don't forget to share about your contribution publicly and provide your support with a star ⭐.

Great work 🔥 !!!!

**Comment by atapas** [comment]
@all-contributors please add @avneesh0612  for Code

**Comment by avneesh0612** [comment]
Awesome thanks @atapas!

**Pull request merged**